op-acceptance-tests/tests/interop/loadtest/interop_load_test.go

```diff
@@ -0,0 +1,60 @@
+package loadtest
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/devnet-sdk/contracts/bindings"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/contracts/constants"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/devstack/devtest"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/devstack/presets"
+	"github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/interop"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/txintent"
+	"github.com/ethereum-optimism/optimism/op-service/txplan"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+var SimpleInterop presets.TestSetup[*presets.SimpleInterop]
+
+// TestMain creates the test-setups against the shared backend
+func TestMain(m *testing.M) {
+	presets.DoMain(m, presets.NewSimpleInterop(&SimpleInterop))
+}
+
+// TODO: this helper is now in at least three places:
+// 1. here
+// 2. op-acceptance-tests/tests/interop/interop_test_helpers.go
+// 3. op-e2e/actions/interop/interop_txplan_test.go
+// Should we make a single helper in devnet-sdk/contracts?
```

Comment:
Question for reviewers. I avoided refactors for the moment but wanted to call this out.